src/Makefile.am

```diff
@@ -1,206 +0,0 @@
-server.c \
-skin.c \
-skin_pcx.c \
-sort.c \
-source.c \
-srv-info.c \
-srv-list.c \
-srv-prop.c \
-stat.c \
-statistics.c \
-utils.c \
-xqf.c \
-xqf-ui.c \
-zipped.c \
-redial.c \
-q3maps.c \
-utmaps.c \
-loadpixmap.c \
-scripts.c \
-addmaster.h \
-addserver.h \
-config.h \
-country-filter.h \
-debug.h \
-dialogs.h \
-dns.h \
-filter.h \
-flt-player.h \
-game.h \
-history.h \
-host.h \
-launch.h \
-menus.h \
-pixmaps.h \
-pref.h \
-psearch.h \
-quake2_pal.h \
-quake_pal.h \
-rc.h \
-rcon.h \
-server.h \
-skin.h \
-skin_pcx.h \
-sort.h \
-source.h \
-srv-info.h \
-srv-list.h \
-srv-prop.h \
-stat.h \
-statistics.h \
-utils.h \
-xqf-ui.h \
-xqf.h \
-zipped.h \
-redial.h \
-q3maps.h \
-utmaps.h \
-loadpixmap.h \
-scripts.h
-
-noinst_SCRIPTS = gensyms.pl
-noinst_PROGRAMS = test_utils
-test_utils_SOURCES = test_utils.c \
-		     utils.c utils.h \
-		     debug.c debug.h
-test_utils_LDADD = $(xqf_LDADD)
-
-# it's only temporary, so ...
-gamesxml2c_LDFLAGS = -lxml2
-gamesxml2c_LDADD = -lxml2
-gamesxml2c_CFLAGS = $(shell xml2-config --cflags)
-
-gamesxml2c_SOURCES = \
-gamesxml2c.c
-
-pkglibexec_PROGRAMS =
-
-if DUMMY_LIBGEOIP
-pkglibexec_PROGRAMS += libxqf_dummy_GeoIP.so.0
-xqf_DEPENDENCIES += libxqf_dummy_GeoIP.so
-endif
-if DUMMY_LIBGDKPIXBUF
-pkglibexec_PROGRAMS += libxqf_dummy_gdk_pixbuf.so.0
-xqf_DEPENDENCIES += libxqf_dummy_gdk_pixbuf.so
-endif
-
-libxqf_dummy_gdk_pixbuf.c:
-	$(top_srcdir)/src/gensyms.pl -x gdk_pixbuf_new_from_file,gdk_pixbuf_render_pixmap_and_mask,gdk_pixbuf_unref $(DUMMY_LIBGDKPIXBUF_ORIG) -o $(top_srcdir)/src/libxqf_dummy_gdk_pixbuf.c || rm $(top_srcdir)/src/libxqf_dummy_gdk_pixbuf.c
-	test -s $(top_srcdir)/src/libxqf_dummy_gdk_pixbuf.c
-
-nodist_libxqf_dummy_gdk_pixbuf_so_0_SOURCES = libxqf_dummy_gdk_pixbuf.c
-libxqf_dummy_gdk_pixbuf_so_0_SOURCES = libxqf_dummy_gdk_pixbuf_stubs.c
-libxqf_dummy_gdk_pixbuf_so_0_LDFLAGS = -nostdlib -shared -Wl,-f,$(DUMMY_LIBGDKPIXBUF_SONAME) -Wl,-soname,libxqf_dummy_gdk_pixbuf.so.0
-libxqf_dummy_gdk_pixbuf_so_0_CFLAGS = -fPIC -DPIC
-
-libxqf_dummy_gdk_pixbuf.so: libxqf_dummy_gdk_pixbuf.so.0
-	rm -f $@
-	$(LN_S) $< $@
-
-libxqf_dummy_GeoIP.c:
-	$(top_srcdir)/src/gensyms.pl -x GeoIP_new,GeoIP_open $(DUMMY_LIBGEOIP_ORIG) -o $(top_srcdir)/src/libxqf_dummy_GeoIP.c || rm $(top_srcdir)/src/libxqf_dummy_GeoIP.c
-	test -s $(top_srcdir)/src/libxqf_dummy_GeoIP.c
-
-nodist_libxqf_dummy_GeoIP_so_0_SOURCES = libxqf_dummy_GeoIP.c
-libxqf_dummy_GeoIP_so_0_SOURCES = libxqf_dummy_GeoIP_stubs.c
-libxqf_dummy_GeoIP_so_0_LDFLAGS = -nostdlib -shared -Wl,-f,$(DUMMY_LIBGEOIP_SONAME) -Wl,-soname,libxqf_dummy_GeoIP.so.0
-libxqf_dummy_GeoIP_so_0_CFLAGS = -fPIC -DPIC
-
-libxqf_dummy_GeoIP.so: libxqf_dummy_GeoIP.so.0
-	rm -f $@
-	$(LN_S) $< $@
-
-CLEANFILES = \
-	$(nodist_libxqf_dummy_gdk_pixbuf_so_0_SOURCES) \
-	$(nodist_libxqf_dummy_GeoIP_so_0_SOURCES) \
-	libxqf_dummy_GeoIP.so \
-	libxqf_dummy_gdk_pixbuf.so
-
-gamelist: gamesxml2c
-	$(top_builddir)/src/gamesxml2c $(top_srcdir)/src/games.xml > $(top_srcdir)/src/games.c
```

Comment:
You know there is a very ugly thing in the code, the `games.c` code is generated from `games.xml` with a `gamesxml2c` that needs to be compiled, we need a way to compile this tool (but not install it).